package/Dockerfile

```diff
@@ -0,0 +1,107 @@
+# Builder image
+FROM registry.suse.com/bci/openjdk:17 AS builder
+
+ARG TARGETOS
+ARG TARGETARCH
+
+COPY . /src
+WORKDIR /src
+
+RUN zypper refresh && \
+    zypper install -y ca-certificates wget curl zip git awk nodejs20 npm20
+
+RUN if [ "$TARGETARCH" = "amd64" ]; then \
+    curl -fL https://github.com/coursier/launchers/raw/master/cs-x86_64-pc-${TARGETOS}.gz | gzip -d > cs; \
+  elif [ "$TARGETARCH" = "arm64" ]; then \
+    curl -fL https://github.com/VirtusLab/coursier-m1/releases/latest/download/cs-aarch64-pc-${TARGETOS}.gz | gzip -d > cs; \
+  else \
+    echo "Unsupported architecture: $ARCH"; \
+    exit 1; \
+  fi && \
+    chmod +x cs && \
+    export PATH="$PATH:/root/.local/share/coursier/bin" && \
+    ./cs install scala:3.3.4 sbt:1.10.2 --install-dir /usr/local/bin
+
+RUN npm install -g @angular/cli@14 && \
+    npm install -g npm-force-resolutions
+
+RUN bash package/build_manager.sh
+
+# Manager unitest
+RUN if [ "$ARCH" = "amd64" ]; then \
+  zypper addrepo https://download.opensuse.org/repositories/M17N:fonts/15.6/M17N:fonts.repo && \
+  zypper --non-interactive --gpg-auto-import-keys refresh && \
```

Comment:
Instead of the `--gpg-auto-import-keys`. The specific repo key can be imported ahead of time, IIRC this should work `rpm --import https://download.opensuse.org/repositories/M17N/15.6/repodata/repomd.xml.key`.